doc/content/source/userobjects/ClusteringUserObject.md

```diff
@@ -0,0 +1,22 @@
+# ClusteringUserObject
+
+## Description
+
+`ClusteringUserObject` works as the base class in Cardinal for indentifying clusters in mesh results, where it may be beneficial to apply
```

Comment:
```suggestion
`ClusteringUserObject` works as the base class in Cardinal for identifying clusters in mesh results, where it may be beneficial to apply
```